### PR TITLE
[SHARE-1009][Improvement] Remove registration form and other unused files

### PR DESCRIPTION
**Pull request by laurenbarker**
## Purpose
Remove registration form and other unused files

## Changes
* Removed a bunch of dead code
* Replaced link in footer with "Aggregated Sources" which is what the status page link replaced before

screen shot 2017-11-30 at 11 34 01 am
screen shot 2017-11-30 at 11 46 53 am

### CODE REVIEW

File: app/templates/profile.hbs [hunk 1]
```diff
@@ -25,6 +15,11 @@
                         <h2>Source registration requests</h2>
                     </div>
                 </div>
+                <p>
+                    We have paused the development of new, SHARE data providers.
```
Comment (aaxelb) [inline app/templates/profile.hbs]: Remove the comma

File: app/templates/registration.hbs [hunk 2]
```diff
@@ -1 +1,7 @@
-{{outlet}}
+<div class="registration-message-container">
+    <h3 class="registration-message">
+        We have paused the development of new, SHARE data providers. Please contact
```
Comment (aaxelb) [inline app/templates/registration.hbs]: bad comma

**Comment by aaxelb** [comment]
We use "source" everywhere else in the interface. Should we say "data sources" for consistency?

**Comment by laurenbarker** [comment]
Yeah, this was taken directly from the ticket but I agree that sources would be better.

**Pull request merged**